ols/src/llms/llm_loader.py

```diff
@@ -67,13 +55,15 @@ def __init__(
         provider: str,
         model: str,
         llm_params: Optional[dict] = None,
+        _llm_providers: Optional[LLMProviders] = None,
+        _llm_providers_reg: Optional[LLMProvidersRegistry] = None,
```

Comment:
I put those as options, but regarding our last discussion @bparees, I can just set those as defaults in init.